### PR TITLE
test formatting + no loc

### PR DESCRIPTION
**Pull request by mairaw**

### CODE REVIEW

File: docs/framework/configure-apps/file-schema/wcf/issuerchannelbehaviors-element.md [hunk 1]
```diff
@@ -2,20 +2,21 @@
...
 
-[**\<configuration>**](../configuration-element.md)\
-&nbsp;&nbsp;[**\<system.serviceModel>**](system-servicemodel.md)\
-&nbsp;&nbsp;&nbsp;&nbsp;[**\<behaviors>**](behaviors.md)\
-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[**\<endpointBehaviors>**](endpointbehaviors.md)\
-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[**\<behavior>**](behavior-of-endpointbehaviors.md)\
-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[**\<clientCredentials>**](clientcredentials.md)\
-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[**\<issuedToken>**](issuedtoken.md)\
-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;**\<issuerChannelBehaviors>**  
+[\<configuration>](../configuration-element.md)\
```
Comment (rpetrusha) [inline docs/framework/configure-apps/file-schema/wcf/issuerchannelbehaviors-element.md]: Although I don't have any strong feeling about this change, it is true that the majority of articles have boldfaced the configuration elements in this section.

**Review by rpetrusha — APPROVED** [review]
LGTM, @mairaw. I've left one comment for you to consider.

File: docs/framework/configure-apps/file-schema/wcf/issuerchannelbehaviors-element.md (hunk 1 above)
Comment (mairaw) [reply docs/framework/configure-apps/file-schema/wcf/issuerchannelbehaviors-element.md]: I think I can do an easy search and replace on existing articles. I was just feeling that the bold was a bit too strong in this section.

**Comment by mairaw** [comment]
@JasonCard can we keep an eye on this topic to see if the terms I'm expecting not to be translated, will indeed be kept in English?

**Comment by JasonCard** [comment]
@mairaw Thank you for letting me know. I'll check the results tomorrow.

**Comment by JasonCard** [comment]
@mairaw [Spanish page](https://docs.microsoft.com/es-es/dotnet/framework/configure-apps/file-schema/wcf/issuerchannelbehaviors-element) was handed back after MT, but the "no-loc" syntax did not work, so most of those lines were MT'ed. 
I found that it didn't work because the content started with escape char like "\\\<behaviors>" instead of "\<behaviors>". It works in most cases if the escape character is removed like "\<behaviors>". However, I also found that it doesn't work for "\<system.serviceModel>" due to the period in the middle (it works without the period). So how about if we try the inline `no-loc` syntax like below?
```
:::no-loc text="[<configuration>](../configuration-element.md)":::\
:::no-loc text="&nbsp;&nbsp;[<systemserviceModel>](system-servicemodel.md)":::\
:::no-loc text="&nbsp;&nbsp;&nbsp;&nbsp;[\<behaviors>](behaviors.md)":::\
:::no-loc text="&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[<endpointBehaviors>](endpointbehaviors.md)":::\
:::no-loc text="&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[<behavior>](behavior-of-endpointbehaviors.md)":::\
:::no-loc text="&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[<clientCredentials>](clientcredentials.md)":::\
:::no-loc text="&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[<issuedToken>](issuedtoken.md)":::\
:::no-loc text="&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<issuerChannelBehaviors>":::
```

**Comment by mairaw** [comment]
I was trying to avoid using the inline version. As we discussed, it would be interesting to learn how delimiters are considered.

**Comment by JasonCard** [comment]
@mairaw Synced with @bodhigautam and for the metadata version of no-loc syntax to work, the word not to be localizable must be surrounded by space. For example, for "<behaviors>" to not be localizable, it cannot be preceded by escape character like "\<behaviors>". So it seems that you will need to use the inline version, but is there a reason why you want to avoid the inline version?

**Pull request merged**